openapi_python_client/parser/properties/model_property.py

```diff
@@ -25,7 +25,6 @@ class ModelProperty(Property):
     discriminator_property: Optional[str]
     discriminator_mappings: Dict[str, Property]
 
-    description: str
```

Comment:
This was a duplicate: ModelProperty is a subclass of Property, which already defines `description`. (This is all specific to the code structure of a very old version of openapi-python-client; the newer versions don't use subclassing.)